packages/cli/src/api/catalog.ts

```diff
@@ -404,6 +404,48 @@ export function getCatalogForFile(file: string, catalogs: Array<Catalog>) {
   return null
 }
 
+/**
+ * Create catalog for merged messages.
+ */
+export function getCatalogForMerge(config: LinguiConfig) {
+  const catalogConfig = config
+
+  if (catalogConfig.mergePath.endsWith(path.sep)) {
```

Comment:
This should fix the appveyor I believe because `path.sep` will be `\` on Windows
```suggestion
  if (catalogConfig.mergePath.endsWith(PATHSEP)) {
```

Reply:
And I think that `normalizeRelativePath(catalogConfig.mergePath)` should be done before this check and work with normalized version.